include/mouse.h

```diff
@@ -136,6 +142,35 @@ bool MOUSEPS2_SendPacket();
 void MOUSEDOS_BeforeNewVideoMode();
 void MOUSEDOS_AfterNewVideoMode(const bool is_mode_changing);
 
+// ***************************************************************************
+// VMM mouse interfaces
+// ***************************************************************************
+
+struct VirtualBoxPointerStatus {
+	uint16_t absolute_x = 0;
+	uint16_t absolute_y = 0;
+};
+
+bool MOUSE_VirtualBox_IsSupported();
+void MOUSE_VirtualBox_ClientDisconnected();
+void MOUSE_VirtualBox_SetPointerAbsolute(const bool is_absolute);
+void MOUSE_VirtualBox_SetPointerVisible(const bool is_visible);
+void MOUSE_VirtualBox_GetPointerStatus(VirtualBoxPointerStatus& status);
+
+struct VMwarePointerStatus {
+	uint16_t absolute_x = 0;
+	uint16_t absolute_y = 0;
+
+	uint8_t buttons       = 0;
+	uint8_t wheel_counter = 0;
+};
+
+bool MOUSE_VMware_IsSupported();
```

Comment:
`VMware` — weird capitalisation in their name, but OK I guess. Worse things have happened in the DOSBox codebase 😅

Reply:
This is the official name capitalisation - see their web site:

![spelling](https://github.com/dosbox-staging/dosbox-staging/assets/48332137/b60992bc-dd7d-4e9b-8f98-a81c9d076e21)

Reply:
Changed capitalization in the source code to `VmWare` (a 'coding rules capitalization'). Comments are still using the 'marketing' capitalization, `VMware` - as this one is used since version 1.0 released in the late 90's.

Reply:
That's for the best, I think.